Recrutify/Recrutify.DataAccess/Repositories/UserRepository.cs

```diff
@@ -15,6 +17,19 @@ public UserRepository(IOptions<MongoSettings> options)
         {
         }
 
+        public async Task<Dictionary<Guid, string>> GetNamesByIdsAsync(IEnumerable<Guid> ids)
+        {
+            var filter = _filterBuilder.In(u => u.Id, ids);
+            var users = await GetCollection().Find(filter).Project(u =>
+                new
+                {
+                    Id = u.Id,
```

Comment:
You can write just:
u.Id,
u.Name,...

Without writing Id=...

Reply:
done